src/main/java/edu/unc/lib/boxc/migration/cdm/InitializeProjectCommand.java

```diff
@@ -87,11 +93,24 @@ public Integer call() throws Exception {
             log.error("Unable to read application configuration", e);
             return 1;
         }
-        var cdmEnvConfig = config.getCdmEnvironments().get(cdmEnvId);
 
         Path currentPath = parentCommand.getWorkingDirectory();
         String projDisplayName = projectName == null ? currentPath.getFileName().toString() : projectName;
+        Integer integer = null;
+
+        if (projectSource.equalsIgnoreCase("cdm")) {
+            integer = initCdmProject(config, currentPath, projDisplayName, start);
+        } else if (projectSource.equalsIgnoreCase("files")) {
+            integer = initFilesProject(currentPath, projDisplayName, start);
+        }
```

Comment:
We can add an `else` and have it log an error message about it being an invalid value (and probably set integer to -1, i'm not really sure what returning a null is going to do).